cmd/makaroni/main.go

```diff
@@ -51,7 +60,7 @@ func main() {
 		IndexHTML:          indexHTML,
 		OutputHTMLPre:      outputPreHTML,
 		Upload:             uploadFunc,
-		ResultURLPrefix:    *resultURLPrefix,
+		ResultURL:          domainUrl.String(),
```

Comment:
It's not the same URL. Imagine if the app is deployed without an L7 router over the app instance and the S3. The app handler could be `https://sub.domain.com/moar/paste` and results are available on the handler like `https://bucket.s3.example.com/dir/{paste_id}`.